zstacklib/tests/test_network_ovs.py

```
@@ -134,3 +135,76 @@ def test_exception_hierarchy(self):
+
+            if "bond_mode" in item:
+                itemDict[item.split(":")[0]] = item.split(":")[1][1:]
+                continue
+
+            if "slave" == item[0:5]:
+                if "slaves" not in itemDict:
+                    slaveList = []
+                slaveDict = {}
+                slaveDict["status"] = item.split(":")[1][1:]
+                slaveDict["name"] = item.split(":")[0].split(" ")[1]
+                pci_addr = eval(shell_call(slaveDict["name"])).strip("\n")
```

Ruff (0.15.9)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)